### ISSUE TITLE
Pre-trained weights are not working

### ISSUE THREAD

**Issue by MuhammadAsadJaved** [tilemmpon/Singapore-Maritime-Dataset-Trained-Deep-Learning-Models#1]
While running the project I used the notebook,  two weights and their corresponding config files  (full_yolo and full_yolo2 )are working right. But using the same procedure SqueezeNet_yolov2 and tiny_yolov2 throw the following error. 
unable to open file : name = 'squeezenet_backend.h5'

and 
unable to open file : name = 'tiny_yolo_backend.h5'

I have tried to use different pats,  download and use weights again but same error. 

Note: it is not path error because if it is path error then it must show the same error for first two models as well.

**Comment by MuhammadAsadJaved** [comment]
I have resolved this issue,      The corresponding backend file should also place in the root folder of the project along with corresponding .json and .h5 file.

**Comment by tilemmpon** [comment]
I am glad you found the solution. Yes, the trained weights (.h5 file in this repository), the pre-trained model (i.e. tiny_yolo_backend.h5),the .config files and the Jupyter notebook must be placed in the same folder (the cloned repository).

**Comment by MuhammadAsadJaved** [comment]
@tilemmpon   BTW do you have the trained faster_rcnn_inception_v2 and ssd_resnet_50 models?   I have used your given four models and weights but it only detects vessel/ship and the accuracy is not the same as your example images in the project.

**Comment by tilemmpon** [comment]
The four models you used were from YOLOv2 architecture and as you can see in the README they perform worse that the one SSD and Faster R-CNN models. The links to all trained models are available in the README. The faster_rcnn_inception_v2 and ssd_resnet_50_fpn models used in the example images were trained using the TensorFlow Object Detection API (see the Repo README for mo details) and their links for reference are:
https://mega.nz/#!euwQSCpT!p6nzkhW73bV7QG7wT15MoljspW1XM3pUZ9djwcgzQkk
https://mega.nz/#!Lq4ygQgB!HNiwxfGyrntAeIHrGGu1kOffxWnwY5LjgjThddbB1-E
For more details about the exact version of TensorFlow used and the respective Jupytet Notebook see the README.

**Comment by MuhammadAsadJaved** [comment]
OK. Bundle of Thanks @tilemmpon

**Comment by MuhammadAsadJaved** [comment]
@tilemmpon  Sir can you provide the 'label_map.pbtxt' used in the notebook for the example images using the Tensorflow Object detection API?

used in this file,  I am unable to find it,  
PATH_TO_LABELS = os.path.join('/home/tbontz2s/git/tensorflow/workspace/training_demo/annotations', 'label_map.pbtxt')

https://github.com/tilemmpon/Singapore-Maritime-Dataset-Trained-Deep-Learning-Models/blob/master/NOTEBOOKS/Tensorflow_object_detection_time.ipynb

**Comment by MuhammadAsadJaved** [comment]
asadjavedgujar@gmail.com

Please send labels file used in TensorFlow notebook

**Comment by tilemmpon** [comment]
It seems that I forgot to include this file, which is essential for Tensorflow to run. I will re-create it and upload soon.

**Comment by MuhammadAsadJaved** [comment]
Thank you very much sir.

**Comment by tilemmpon** [comment]
The label map can be found in https://github.com/tilemmpon/Singapore-Maritime-Dataset-Trained-Deep-Learning-Models/blob/master/tensorflow_training_configurations/label_map/label_map.pbtxt . I re-created it and it should work.

**Comment by MuhammadAsadJaved** [comment]
@tilemmpon  Sir this tensorFlow notebook is only to calculate processing time? How i can save the detected images same as we saved in YOLO and can we run it on videos as well?  if so then can you provide the script you used to save the output images and videos or I need to write a script by myself?

**Comment by tilemmpon** [comment]
The scripts are provided in the README in the section of the example inferred images. I used an adapted version of the tensorflow tutorial (https://github.com/tensorflow/models/blob/master/research/object_detection/object_detection_tutorial.ipynb) to get results. For video, I did not write any script. You can try to write one yourself.

**Comment by MuhammadAsadJaved** [comment]
@tilemmpon . Thanks sir I got it.  I have another question.  I want to train YOLOV3 model with caffe implementation. Can you provide the script or link used to generate images and related annotation  .txt or .xml files for YOLOV2 training?  Because for YOLO model training we need dataset as images and corresponding .txt or .xml file and then convert thme to YOLO formate. 

https://github.com/tilemmpon/Singapore-Maritime-Dataset-Frames-Ground-Truth-Generation-and-Statistics/blob/master/Singapore_dataset_frames_generation_and_histograms.ipynb

I saw this script to convert videos to images to create dataset but i did not find the script which you used to generate new annotations for generated images.  Can you upload the images and corresponding annotations files?   I can contribute my trained models on YOLOV3 to this repo. 

Thank you very much.

**Comment by MuhammadAsadJaved** [comment]
got it.  @tilemmpon  Thank you sir.
